packages/@o3r/core/schematics/ng-add/project-setup/index.ts

```diff
@@ -40,54 +41,71 @@ export const prepareProject = (options: NgAddSchematicsSchema) => async (tree: T
-    ...(options.enableAnalytics ? ['@o3r/analytics'] : []),
-    ...(options.enableApisManager ? ['@o3r/apis-manager'] : []),
     ...(options.enableStorybook ? ['@o3r/storybook'] : []),
-    ...(options.enablePlaywright ? ['@o3r/testing'] : []),
-    ...(options.enableRulesEngine ? ['@o3r/rules-engine'] : []),
     ...(installOtterLinter ? ['@o3r/eslint-config-otter'] : [])
   ]));
   const projectPackageJson = workspaceProject ? readPackageJson(tree, workspaceProject) : tree.readJson('package.json') as PackageJson;
+  const externalPackagesToInstallWithNgAdd: Record<string, string | undefined> = {};
   // Ngx-prefetch version is aligned with angular
-  const angularVersion = projectPackageJson.dependencies?.['@angular/core'] || projectPackageJson.peerDependencies?.['@angular/core'];
-  const angularMajorVersion = angularVersion?.match(simplifiedSemVerRegexp)?.[1];
-  const ngxPrefetchVersion = angularMajorVersion ? `^${angularMajorVersion}.0.0` : undefined;
-  const type = projectType === 'application' ? NodeDependencyType.Default : NodeDependencyType.Peer;
-  const externalPackagesToInstallWithNgAdd = Array.from(new Set([
-    ...(options.enablePrefetchBuilder ? ['@o3r/ngx-prefetch'] : [])
-  ]));
-  return () => chain([
+  if (projectType === 'application' && options.enablePrefetchBuilder) {
+    const angularVersion = projectPackageJson.dependencies?.['@angular/core'] || projectPackageJson.peerDependencies?.['@angular/core'];
+    const angularMajorVersion = angularVersion?.match(simplifiedSemVerRegexp)?.[1];
+    const ngxPrefetchVersion = angularMajorVersion ? `^${angularMajorVersion}.0.0` : undefined;
+    externalPackagesToInstallWithNgAdd['@o3r/ngx-prefetch'] = ngxPrefetchVersion;
+  }
+  const type = projectType === 'library' ? NodeDependencyType.Peer : NodeDependencyType.Default;
+  let appLibRules: Rule[] = [];
+  let monorepoRules: Rule[] = [];
+  if (projectType) {// this means we run the ng add in a lib/app inside the workspace
+    appLibRules = [
+      updateImports(mapImportV7toV8, renamedPackagesV7toV8) as any,
+      updateBuildersNames(),
+      updateOtterGeneratorsNames(),
+      // TODO: split `updateCmsAdapter` function do be handle by each module ng-add schematic
+      // options.enableCms ? updateCmsAdapter(options, coreSchematicsFolder) : noop,
+      updateOtterEnvironmentAdapter(options, coreSchematicsFolder),
+      updateStore(options, projectType),
+      options.enableCustomization && projectType === 'application' ? updateCustomizationEnvironment(coreSchematicsFolder, o3rCoreVersion, options, false) : noop,
+      options.enablePlaywright && projectType === 'application' ? updatePlaywright(coreSchematicsFolder, options) : noop,
+      projectType === 'application' ? updateAdditionalModules(options, coreSchematicsFolder) : noop,
+      removePackages(packagesToRemove)
+    ];
+  } else {
+    monorepoRules = [
+      generateRenovateConfig(coreSchematicsFolder),
```

Comment:
no renovate on the applications?

Reply:
This is tricky good point. The reco what we'll put in place will be to have a always  monorepo even if you have only one app. This will prepare the things in case you want to add a new app or a new lib (the structure will be ready). For this we will setup renovate at monorepo level.
But if we run the ng add on an existing repository which is only an app for instance, we should add it there too. 
So I have to identify somehow that I am in app but not in a monorepo :S.
cc: @kpanot